pkgs/tools/text/ansifilter/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, pkgs }:
+let version = "1.15";
+    pkgsha = "65dc20cc1a03d4feba990f830186404c90462d599e5f4b37610d4d822d67aec4";
+in with pkgs; stdenv.mkDerivation {
+  name = "ansifilter";
+  buildInputs = [
+    stdenv gcc gnumake
+    pkgconfig boost lua
+  ];
+  src = fetchurl {
+    url = "http://www.andre-simon.de/zip/ansifilter-${version}.tar.bz2";
+    sha256 = pkgsha;
+  };
+  meta = {
+    description = "tool";
+  };
+
+  configurePhase = ''makeFlags="PREFIX=$out conf_dir=$out/etc/ansifilter/"'';
```

Comment:
You can provide additional make flags by specifying a `makeFlags` attribute, as in `makeFlags = "conf_dir=$(out)/etc/ansifilter"`.